net/http/inc/THttpLongPollEngine.h

```diff
@@ -15,15 +15,34 @@
 #include "THttpWSEngine.h"
 
 #include <string>
-#include <vector>
+#include <list>
+
+class THttpServer;
 
 class THttpLongPollEngine : public THttpWSEngine {
+friend class THttpServer;
 protected:
-   THttpCallArg *fPoll;              ///!< polling request, which can be used for the next sending
-   std::vector<std::string> fQueue;  ///!< entries submitted to client
+   class QueueItem {
+   public:
+      const void *buf{nullptr}; ///<! raw memory with ownership
```

Comment:
Then please use a `unique_ptr`, ideally even of a `vector<char>` to kill the `len` member.